website/src/views/DashboardHome/DashboardHome.jsx

```diff
@@ -76,14 +94,18 @@ class DashboardHome extends PureComponent {
             eventName: 'Outline of Research Paper',
             dueTime: '6:30pm',
             color: '#62c4f9',
+            key: 7,
+
           },
         ],
+        key: 'Oct. 31',
+
       },
     ];
     return (
       <div className="DashboardHome">
         {classArray.map(event => (
-          <EventSummaryCollection event={event} />
+          <EventSummaryCollection key={event.key} event={event} />
```

Comment:
`key` here should be the unique ID generated in our backend. Will let you know more about this as the events api is completed.

Reply:
Cool as of now i based this off of the temp schema when you complete the events api i can update it to support a new schema